### DISCUSSION TITLE
Possible inconsistency with ffmpeg 4.4

### DISCUSSION THREAD

**Discussion by klassikmann** [rern/rAudio#312] [Fixed Issues]
**rAudio 1 • 20210416
5.10.32-1-ARCH aarch64**

Today's "pacman -Syu" installed ffmpeg 4.4. Playing a web-radio requiring ffmpeg (= my transport streams provided by the SAT-receiver) is successful but: The web-radio doesn't appear in the playlist but can be deleted from it.

**Comment by klassikmann** [comment]
Still present in rAudio 1 • 20210428

**Comment by klassikmann** [comment]
Still present in rAudio 1 • 20210502

**Reply by klassikmann** [reply]
Today a "pacman -Syu" installed a new ffmpeg package: ffmpeg 2:4.4-2
Problem persists.

**Reply by rern** [reply]
Try `mpc add WEBRADIO_URL` to see errors in output.

**Reply by klassikmann** [reply]
Still present in rAudio 1 • 20210507

**Reply by klassikmann** [reply]
@rern 

Still present in rAudio 1 • 20210515
It's indeed an unimportant issue. But if you need assistance from my side (logs or whatever) please let me know.

**Reply by R101G** [reply]
@klassikmann 
If I understand your report correctly, I don't see this. When I play an m3u8 stream it appears in the playlist. I do have "Tap Song = Replace And Play" enabled. Are you playing mp2 streams?

**Reply by klassikmann** [reply]
@R101-Github 

> Are you playing mp2 streams?
Yes. MP2 within a MPEG2 TS.

**Reply by R101G** [reply]
Far too complicated for me, but wouldn't that be decoded by lame rather than ffmpeg?

**Reply by klassikmann** [reply]
@R101-Github 

> Far too complicated for me, but wouldn't that be decoded by lame rather than ffmpeg?

ffmpeg is needed to demux the TS-Stream finding an appropriate elementary stream, here MP2.
This procedure works now as it worked in the past with the one difference, that the playlist seems to be empty in the GUI, but it isn't, because "Remove from Playlist"->"All"  removes that webradio from the playback window (and stops the webradio before, if the webradio is still playing).

**Comment by klassikmann** [comment]
After a month I was looking again after the issue in **rAudio 1 • 202106191**.
It has been fixed in the meantime.
